liquibase.gradle

```diff
@@ -3,7 +3,7 @@ configurations {
 }
 
```

Comment:
in the previous PR we introduced db.changelog-RDM-9052.xml
Are we able to get rid of it now after the update to the newest liquibase version or we still need that?

Reply:
We got rid of it.